readme.md

```diff
@@ -48,4 +55,36 @@ String[] arguments = new String[] {
 org.apache.sling.feature.launcher.impl.Main.main(arguments);
 ```
 
+# Container
+
+The container-image `apache/sling-org-apache-sling-feature-launcher:latest` is avaiable on DockerHub.
+
+Available tags:
+- **latest**    - latest build, could be a `snapshot` **or** `release` version.
+- **snapshot**  - latest build of a `*-SNAPSHOT` version.
+- **stable**    - latest build of a **release** version.
+- **(version)** - latest build of a fixed version, could be a `snapshot - (1.1.9-SNAPSHOT)` **or** `release - (1.1.9)` version.
```

Comment:
No snapshots please

Reply:
Could you be more explicit.

Will we do this only on creating a git tag?
This should be then tagged as:
-latest
-tagname
- git hash ?

No snapshot!
No stable?

Reply:
For more context - the ASF release policy says that we only publish releases, not CI artifacts. So for DockerHub, which is a public repository, we should only publish for git tags. A separate discussion is to maybe generate them manually, since after a tag we have a release vote, and only them the publication should happen.

So we should have:
- version - e.g. `1.0.20`
- latest

I don't think git hash is useful so I would not use that.